modules/L10_system_emulation/run_service.sh

```diff
@@ -27,7 +27,7 @@ if ("${FIRMAE_ETC}"); then
       BINARY_NAME=$("${BUSYBOX}" basename "${BINARY_NAME}")
       if ( ! ("${BUSYBOX}" ps | "${BUSYBOX}" grep -v grep | "${BUSYBOX}" grep -sqi "${BINARY_NAME}") ); then
         "${BUSYBOX}" echo "[*] Starting $BINARY_NAME service ..."
-        ${BINARY} &
+        "${BINARY}" &
```

Comment:
in this BINARY variable could be something like:
`binary parameter parameter ...`
In this special case we can't add the "" to it.
Please, could you add a comment to ensure we do not forget about this situation?

Reply:
Ok, but I am renaming the variable to _BINARY, so semgrep won't flag it as an unquoted variable.